scripts/deployment/test_model_path_detection.py

```
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+"""
+🧪 Test Model Path Detection
+============================
+Test the portable model directory detection logic.
+"""
+
+import os
+import sys
+
+# Add the upload script to path to import the function
+script_dir = os.path.dirname(os.path.abspath(__file__))
```

Ruff (0.12.2)

`os.path.abspath()` should be replaced by `Path.resolve()`

(PTH100)